docs/reference/synonyms/apis/synonyms-apis.asciidoc

```diff
@@ -29,8 +34,25 @@ Use the following APIs to manage synonyms sets:
 * <<list-synonyms-sets>>
 * <<delete-synonyms-set>>
 
+[discrete]
+[[synonym-rules-apis]]
+=== Synonym rules APIs
+
+Synonyms sets are composed of synonym rules.
+Each synonym rule defines a group of words that act as synonyms.
```

Comment:
I feel like this could be clearer...

Do we want to explain what format the rules come in? I understand that there are 2 flavors:

- equivalency (where terms are equal substitutes in the query)
- explicit mappings (where terms are mapped to one explicit term prior to querying)

Reply:
We can do that. I was thinking on providing a specific synonyms guide that would explain the format (as per this proposal) and other places would link to it.

Take into account that we're describing the format in both the [synonym](https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-synonym-tokenfilter.html) and [synonym graph](https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-synonym-graph-tokenfilter.html) token filter. It would be nice to do so in a single place IMO.

Reply:
Sounds good